geopandas/tests/test_cmapplotting.py

```diff
@@ -0,0 +1,271 @@
+    MPL_DFT_COLOR = cycle['color'][0]
+except KeyError:
+    MPL_DFT_COLOR = matplotlib.rcParams['axes.color_cycle'][0]
+
+class TestCMapPointPlotting:
+    def setup_method(self):
+        # Intializing the dataframe
+        self.N = 10
+        self.points = GeoSeries(Point(i, i) for i in range(self.N))
+        self.values = np.arange(self.N)
+        self.df = GeoDataFrame({'geometry': self.points, 'values': self.values})
+
+    def test_customnorm(self):
+        # test color without additional parameters
+        from geopandas.plotting import plot_point_collection
+        fig, ax = plt.subplots()
+        coll = plot_point_collection(ax, self.points, self.values)
+        fig.canvas.draw_idle()
+
+        cmap = plt.get_cmap()
+        expected_colors = cmap(np.arange(self.N) / (self.N - 1))
+        np.testing.assert_array_equal(coll.get_facecolors(), expected_colors)
+        ax.cla()
+
+        fig, ax = plt.subplots()
+        cmap = mpl.cm.Greys
+        norm = mpl.colors.BoundaryNorm([1,3,6,8], cmap.N)
+        coll = plot_point_collection(ax, self.points, self.values, norm = norm)
+        fig.canvas.draw_idle()
+
+        cmap = plt.get_cmap()
+        expected_colors = cmap(norm(list(self.df['values'])))
+        np.testing.assert_array_equal(coll.get_facecolors(), expected_colors)
+        ax.cla()
+
+    def test_minmax(self):
+        # test min-max without additional parameters
+        from geopandas.plotting import plot_point_collection
+        fig, ax = plt.subplots()
+        coll = plot_point_collection(ax, self.points, self.values)
+        fig.canvas.draw_idle()
+        actual_lim = coll.get_clim()
+        assert actual_lim == (0,9)
+        ax.cla()
+
+        fig, ax = plt.subplots()
+        cmap = mpl.cm.Greys
+        norm = mpl.colors.BoundaryNorm([1,3,6,8], cmap.N)
+        coll = plot_point_collection(ax, self.points, self.values, norm = norm)
+        fig.canvas.draw_idle()
+        actual_lim = coll.get_clim()
+        assert actual_lim == (1,8)
+        ax.cla()
+
+    def test_subset_customnorm(self):
+        from geopandas.plotting import plot_point_collection
+        fig, ax = plt.subplots()
+
+        cmap = mpl.cm.Greys
+        norm = mpl.colors.BoundaryNorm([1,2,4], cmap.N)
+        coll = plot_point_collection(ax, self.points[0:3], self.values[0:3], norm = norm)
+        fig.canvas.draw_idle()
+
+        cmap = plt.get_cmap()
+        expected_color = cmap(np.arange(self.N) / (self.N - 1))
+
+        np.testing.assert_array_equal(coll.get_facecolors()[0], expected_color[0])
+        np.testing.assert_array_equal(coll.get_facecolors()[-1], expected_color[-1])
+
+        ax.cla()
+
+    def test_subset_minmax(self):
+        from geopandas.plotting import plot_point_collection
+        fig, ax = plt.subplots()
+
+        cmap = mpl.cm.Greys
+        norm = mpl.colors.BoundaryNorm([1,3,6,8], cmap.N)
+        coll = plot_point_collection(ax, self.points, self.values, norm = norm)
```

Comment:
again, you're not passing in the cmap despite setting it. I'm not sure how this test passes given the cmap here would I think be viridis.